test/test_app_logging.py

```diff
@@ -73,7 +73,8 @@ def fail():
 
                     # The exception is always logged
                     self.assertEqual(len(app_log.output), 1)
-                    err_log = 'ERROR:test_app_logging:Caught exception for <function TestAppLogging.test.<locals>.fail'
+                    err_log = 'ERROR:test_app_logging:Caught exception for ' \
+                              '<function AzulChaliceApp.route.<locals>.decorator.<locals>.f'
```

Comment:
```suggestion
                    err_log = 'ERROR:test_app_logging:Caught exception for ' \
                              '<function AzulChaliceApp.route.<locals>.decorator.<locals>.fail'
```

Reply:
`f` is from the method inside the decorator in `src/azul/Chalice.py` ln 82.